plugin/plugin.go

```diff
@@ -501,7 +515,12 @@ func (p *plugin) generateFloatValidator(variableName string, ccTypeName string,
 }
 
 func (p *plugin) generateStringValidator(variableName string, ccTypeName string, fieldName string, fv *validator.FieldValidator) {
-	if fv.Regex != nil {
+	if fv.Regex != nil || fv.GetUuidVer() != 0 {
```

Comment:
You can almost copy-paste the already modified logic from the `generateRegexVar` function above. No need to refactor into a separate function (yet) to keep the code flow localised for improved readability.

Reply:
I'm not entirely sure if I got the idea.
Please have a look again, I aligned the logic to be more then then one in `generateRegexVar()`